Example/testComponents/CleanUpTest.js

```diff
@@ -0,0 +1,228 @@
+import React from 'react';
+import Animated, { useSharedValue, useWorklet, RegistersState } from 'react-native-reanimated';
+import { View, Text } from 'react-native';
+
+const workletBody = function(a, b, c) {
```

Comment:
If you don't create Worklet instance here then each Event Worklet will register separate Worklet. Is this really what you want?

Reply:
yes this is it exactly